### PR TITLE
postgresqlPackages.timescaledb: 2.5.2 -> 2.6.0

### PR DESCRIPTION
**Pull request by 1000101**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/sql/postgresql/ext/timescaledb.nix [hunk 1]
```diff
@@ -8,17 +8,17 @@
...
 
   src = fetchFromGitHub {
-    owner  = "timescale";
-    repo   = "timescaledb";
+    owner = "timescale";
+    repo = "timescaledb";
     # some branches are named like tags which confuses git
-    rev    = "refs/tags/${version}";
-    sha256 = "sha256-eq2dljS8+0fRcA7hQr5yOV1FXtMyaajjYFqpw+DYt1A=";
+    rev = "refs/tags/${version}";
```
Comment (ajs124) [inline pkgs/servers/sql/postgresql/ext/timescaledb.nix:20]: doesn't `rev = version;` work the same?

Comment (marsam) [reply pkgs/servers/sql/postgresql/ext/timescaledb.nix:20]: Sorry about that! I merged this before my morning coffee
>doesn't rev = version; work the same?

yes, it works the same. I'll address it in the next bump.

Comment (ajs124) [reply pkgs/servers/sql/postgresql/ext/timescaledb.nix:20]: no problem, and sorry for being so confrontational about it.

there seem to be a bunch (around 100) of packages using `refs/tags`, maybe we should change them all, assuming there is actually no good reason to do that.
also the documentation (doc/builders/fetchers.chapter.md) should maybe be rephrased.

Comment (1000101) [reply pkgs/servers/sql/postgresql/ext/timescaledb.nix:20]: @ajs124 sorry for not responding in the first place! I was waiting on further comments :) ...I'll change it in the next update (if it doesn't get updated in the meantime with the other pkgs).

**Pull request merged**